src/main/java/com/github/shiranr/scooters/api/Admin.java

```diff
@@ -38,14 +49,35 @@ public HttpResponseMessage history(
      */
     @FunctionName("create")
     public HttpResponseMessage create(
-            @HttpTrigger(name = "create",
+            @HttpTrigger(
+                    route = "scooter/{amount}",
+                    name = "history",
                     methods = {HttpMethod.POST},
-                    authLevel = AuthorizationLevel.ANONYMOUS,
-                    route = "scooter/create/{amount}")
+                    authLevel = AuthorizationLevel.ADMIN)
                     HttpRequestMessage<Optional<String>> request,
             @BindingName("amount") int amount,
             final ExecutionContext context) {
-        //TODO implement - committing only stubs. Next iteration will include implementation.
-        return request.createResponseBuilder(HttpStatus.OK).body(amount).build();
+        IntStream.range(0, amount).mapToObj(i -> random()).forEachOrdered(id -> {
```

Comment:
Can you please leave a comment here and explain what is this line doing?
Are you using the random helper methods you created?

Reply:
Will do it on next commit.